includes/Admin/Admin_Page.php

```diff
@@ -203,4 +215,15 @@ public function admin_footer() {
 			)
 		);
 	}
+
+	/**
+	 * Get hook_suffix property.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string
```

Comment:
```suggestion
	/**
	 * Gets the hook suffix under which the admin page is added.
	 *
	 * @since n.e.x.t
	 *
	 * @return string Hook suffix, or empty string if admin page was not added.
```